taxonomies/Message.ttl

```diff
@@ -385,6 +385,51 @@ ids:LogMessage a owl:Class;
     rdfs:label "Log Message"@en ;
     rdfs:comment "Log Message which can be used to transfer logs e.g., to the clearing house."@en.
 
+ids:LogEntry a owl:Class;
+    rdfs:label "Log Entry"@en ;
+    rdfs:comment "Contains the metadata of a LogMessage"@en .
+
+ids:transactionID a owl:DatatypeProperty ;
+    rdfs:domain ids:LogEntry ;
+    rdfs:range xsd:unsignedInt ;
+    rdfs:comment "Links a LogEntry (Metadata) to a transactionID"@en .
```

Comment:
@PHochmann I suggest to add `rdfs:label` to all newly added properties. That would basically be readable versions of their URIs, like `transactionID` -> `transaction id/ID`